autometa/taxonomy/ncbi.py

```diff
@@ -202,62 +206,56 @@ def get_lineage_dataframe(self, taxids, fillna=True):
                 else:
                     ranked_taxids[taxid].update({rank: name})
         df = pd.DataFrame(ranked_taxids).transpose()
-        df.index.name = 'taxid'
+        df.index.name = "taxid"
         if fillna:
-            df.fillna(value='unclassified', inplace=True)
+            df.fillna(value="unclassified", inplace=True)
         return df
 
     def rank(self, taxid):
         """
-        Takes a taxid and returns its rank
+        Return the respective rank of provided `taxid`. If the `taxid` is deprecated, suppressed,
+        withdrawn from NCBI (basically old) the updated rank will be retrieved
 
         Parameters
         ----------
         taxid : int
-            identifer for a taxon in the Taxonomy Database by NCBI
-            `taxid` to retrieve rank from `nodes` or `merged`
+            `taxid` to retrieve rank from nodes.dmp
 
         Returns
         -------
         str
-            rank name if taxid is found in nodes.dmp or merged.dmp else 'unclassified'
+            rank name if taxid is found in nodes.dmp else "unclassified"
```

Comment:
In this part I don't know why you removed reference to `merged.dmp` because the top part of the docstring implies that `merged.dmp` will be checked if the `taxid` is deprecated.